### PR TITLE
Grammatical fixes

### LINKED ISSUE #54: Spelling, punctuation and grammar screening
## Acceptance Criteria
- [x] Check punctuation is correct on the home page P1 & P2
- [x] Check the punctuation is correct in info3 and info5 content
- [x] Check no double "the" in "the The Advocacy"

### PR DESCRIPTION
**Pull request by j-gynn**
Fixes general SPaG errors

@neontribe/ca-seap

### CODE REVIEW

**Review by rosejbon — APPROVED** [review]
Thanks Jess, looks good :)

File: src/Copy/BrandCopy.elm [hunk 1]
```diff
@@ -121,12 +121,12 @@ brandCopy key =
...
+                , CopyText "If there are parts of your complaint you cannot do for yourself, you can ask for one of The Advocacy People's team to be your advocate. They will listen to you, support you to speak for yourself, and help you to make choices. They can also write letters for you, help you in complaint meetings, and talk to other people for you if necessary."
                 ]
 
         HomeP2 ->
-            CopyText "The The Advocacy People service is completely confidential, free and independent of the NHS. It can only be used to complain about NHS services. We can’t help with complaints about private health care or social care provided by the local council."
+            CopyText "The Advocacy People's service is completely confidential, free and independent of the NHS. It can only be used to complain about NHS services. We can’t help with complaints about private health care or social care provided by the local council."
```
Comment (katjam) [inline src/Copy/BrandCopy.elm:129]: @roseb04 What do you think? I'm not sure on this.

File: src/Copy/BrandCopy.elm [hunk 2]
```diff
@@ -121,12 +121,12 @@ brandCopy key =
...
-                , CopyText "If there are parts of your complaint you cannot do for yourself, you can ask for one of the The Advocacy People team to be your advocate. They will listen to you, support you to speak for yourself, and help you to make choices. They can also write letters for you, help you in complaint meetings, and talk to other people for you if necessary."
+                [ CopyText "When something goes wrong and you feel like the NHS has let you down, it can be hard. You have a right to complain and the NHS needs your feedback in order to improve their service. Call The Advocacy People for free information about getting your voice heard and other organisations that can help."
+                , CopyText "If there are parts of your complaint you cannot do for yourself, you can ask for one of The Advocacy People's team to be your advocate. They will listen to you, support you to speak for yourself, and help you to make choices. They can also write letters for you, help you in complaint meetings, and talk to other people for you if necessary."
```
Comment (katjam) [inline src/Copy/BrandCopy.elm:125]: Not sure if it should be `'s` Does their own website give any clues?

**Review by katjam — COMMENTED** [review]
I'd like @roseb04 or @katquatermass to double check the grammatical copy changes since they `own` the copy... but looks pretty good to me.

File: src/Copy/BrandCopy.elm (hunk 2 above)
Comment (j-gynn) [reply src/Copy/BrandCopy.elm:125]: From a readability standpoint, I would advocate for this since it makes the sentence "flow" better. Just my two cents.

Comment (rosejbon) [reply src/Copy/BrandCopy.elm:125]: I'm ok with it being written either way to be honest, so happy to keep Jess's suggestion if she feels the sentence flows better this way. 

I have seen many charities write it both ways.

**Comment by katjam** [comment]
@RabidGamer looks like rose is happy with this. Fix the conflicts and should be good to go.

If you tag it, then check on staging after - we can get this pushed live today.

**Pull request merged**